tests/sentry/backup/test_correctness.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from difflib import unified_diff
+from pathlib import Path
+from typing import NamedTuple
+
+import pytest
+from click.testing import CliRunner
+from freezegun import freeze_time
+
+from sentry.db.postgres.roles import in_test_psql_role_override
+from sentry.runner.commands.backup import export, import_
+from sentry.testutils.factories import get_fixture_path
+from sentry.utils import json
+from sentry.utils.json import JSONData, JSONEncoder, better_default_encoder
+
+
+# TODO(team-ospo/#155): Figure out if we are going to use `pk` as part of the identifier, or some other kind of sequence number internal to the JSON export instead.
```

Comment:
in the future, these will be moved into perhaps the backup file, correct?

Reply:
Yep, working on that change now! :)